commons-geometry-euclidean/src/main/java/org/apache/commons/geometry/euclidean/PointMap2DImpl.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.geometry.euclidean;
+
+import java.util.List;
+
+import org.apache.commons.geometry.core.collection.PointMap;
+import org.apache.commons.geometry.core.internal.AbstractBucketPointMap;
+import org.apache.commons.geometry.euclidean.twod.Vector2D;
+import org.apache.commons.numbers.core.Precision;
+
+/** Internal {@link PointMap2D} implementation.
```

Comment:
@darkma773r I have a project in QGIS now, and am trying to play with the code with the data that I have. I opened `PointMap` and `PointSet` in Eclipse, to look at the implementations. Then found `PointMap2DImpl`, but this link above, to `PointMap2D` is not working in Eclipse… I searched for `PointMap2D` here too, and couldn't locate it? I think `PointMap2S` has a similar link that's not working in Eclipse, so maybe it occurs elsewhere as well.

Reply:
Darn. Those were extensions of `PointMap` specific to those spaces. I ended up removing them since they didn't really add anything. I thought I removed all of the references in the docs but I guess not.